src/DataSet/ObjectDataSet.php

```diff
@@ -54,16 +123,14 @@ public function getAttributeValue(string $attribute): mixed
             return $this->object->getAttributeValue($attribute);
         }
 
-        return $this->hasAttribute($attribute)
-            ? $this->reflectionProperties[$attribute]->getValue($this->object)
-            : null;
+        return $this->getData()[$attribute] ?? null;
```

Comment:
```suggestion
        return ($this->getReflectionProperties()[$attribute] ?? null)?->getValue($this->getObject());
```

Reply:
Not need get all data when receiving one attribute value.

Reply:
Good catch. :+1: It was left since time when the data was cached.